### PR TITLE
[ENG-599] Fail tenderly cli if there is .ts files and no tsconfig in sources dir

### PR DESCRIPTION
**Pull request by veljko-matic**

### CODE REVIEW

File: commands/actions/publish.go [hunk 1]
```diff
@@ -95,8 +95,20 @@ func buildFunc(cmd *cobra.Command, args []string) {
 	}
 	mustParseAndValidateTriggers(actions)
 
+	existsTsFiles, err := util.TsFilesExists(actions.Sources)
+	if err != nil {
+		logrus.Error(commands.Colorizer.Sprintf("Found error when finding *.ts files, Error: %s", commands.Colorizer.Bold(commands.Colorizer.Red(err))))
+		os.Exit(1)
```
Comment (delta003) [inline commands/actions/publish.go:101]: This is an optional, best effort check, I'd consider ignoring error here, just logging skip the check

File: commands/actions/publish.go [hunk 2]
```diff
@@ -95,8 +95,19 @@ func buildFunc(cmd *cobra.Command, args []string) {
 	}
 	mustParseAndValidateTriggers(actions)
 
+	existsTsFiles, err := util.TsFilesExists(actions.Sources)
+	if err != nil {
+		logrus.Info("\nFound error when finding *.ts files")
```
Comment (delta003) [inline commands/actions/publish.go:100]: Let's log WARN at least

File: commands/actions/publish.go [hunk 3]
```diff
@@ -95,8 +95,19 @@ func buildFunc(cmd *cobra.Command, args []string) {
...
+	}
+	tsConfigExists := util.TsConfigExists(actions.Sources)
+
+	if existsTsFiles && !tsConfigExists {
+		logrus.Error(commands.Colorizer.Bold(commands.Colorizer.Red(commands.Colorizer.Sprintf("Missing typescript config file in your sources! Sources: %s", actions.Sources))))
```
Comment (delta003) [inline commands/actions/publish.go:105]: I think we do something like

```
userError.LogErrorf("failed parsing action trigger with %s",
				userError.NewUserError(err,
					commands.Colorizer.Sprintf(
						"Failed parsing action trigger for %s",
						commands.Colorizer.Bold(commands.Colorizer.Red(name))),
				),
			)
			os.Exit(1)
```

**Review by delta003 — CHANGES_REQUESTED** [review]

**Review by delta003 — COMMENTED** [review]
LGTM, deferring to Sloba to take a look and approve

**Comment by sl0b0dan** [comment]
LGTM

**Review by sl0b0dan — APPROVED** [review]

**Pull request merged**